fastlane_core/lib/fastlane_core/configuration/configuration.rb

```diff
@@ -171,8 +171,28 @@ def load_configuration_file(config_file_name = nil, block_for_missing = nil, ski
       return if paths.count == 0
 
       path = paths.first
-      configuration_file = ConfigurationFile.new(self, path, block_for_missing, skip_printing_values)
+      begin
```

Comment:
I implemented it this way so that it would pass the test [here](https://github.com/fastlane/fastlane/blob/master/fastlane_core/spec/configuration_file_spec.rb#L201)... I don't think it's actually a good idea though, since it basically crashes anyway after that point. Could any actual setups use this "feature" or could it be removed entirely?